fr-index.html

```diff
@@ -0,0 +1,56 @@
+---
+title: "Jamulus ‒ Jouez de la musique en ligne. Avec vos amis. Gratuitement."
+lang: "fr"
+permalink: /
+layout: mainhomepage
+ldjAppCategory: "Musique"
+ldjSlogan: "Jouez de la musique en ligne. Avec vos amis. Gratuitement."
+ldjCreator: "Volker Fischer et divers contributeurs"
```

Comment:
I think « et contributeurs »  is enough.
Or if we want take care about womens « et contributeurs et contributrices » (no inclusive writing).
« divers » sounds with a pejorative meaning to me.